src/ui/Roster.ts

```diff
@@ -79,6 +84,78 @@ export default class Roster {
          spanElement.text(msg);
       }
    }
+   
+   private initCollapisible()
+   {
+      let coll = $(".jsxc-collapsible");
+      let i;
+
+      for (i = 0; i < coll.length; i++)
+      {
+         $(coll[i]).off("click").on("click", function()
+         {
+            this.classList.toggle("jsxc-active");
+            let content = $(this).next();
+            if ($($(content)[0]).css('display')!=='none'){
+                $($(content)[0]).css('display','none');
+                $($(content)[0]).css('max-height','');
+            } else {
+                $($(content)[0]).css('display','');
+                $($(content)[0]).css('max-height','100%');
+            }
+         });
+
+         $(coll[i]).next().css('flex-grow', 'unset');
+      }
+
+      if (this.contactList[0].childNodes.length>0)
+      {
+          $(this.contactList[0]).css('display','');
+          $(this.contactList[0]).css('max-height','100%');
+      }
+   }
+   
+   private addContactFilter()
+   {
+      $("#filterlist").off("keyup").on("keyup",function(e)
+      {
+         $.each( $(".jsxc-contact-list").children("li"), function( key, value )
+         {
+             $(this).css({"display":""});
+         });
+         $.each( $(".jsxc-group-list").children("li"), function( key, value )
+         {
+             $(this).css({"display":""});
+         });
+         
+         let val = null;
+         
+         if ($(this).val())
+            val = new String($(this).val());
+         
+         if (val!=null&&val.length>0)
```

Comment:
We should use type comparison `!==`. Also this check should not be required, because the val is evaluated before.

Reply:
now using $(this).val().toString()...